_data/osdf_namespace_metadata/igwn-shared.yaml

```diff
@@ -1,12 +1,20 @@
 description: |
+  Curated datasets used by members of the [LIGO Scientific Collaboration](www.ligo.org), the
+  [Virgo Collaboration](https://www.virgo-gw.eu/), and the [KAGRA Collaboration](https://gwcenter.icrr.u-tokyo.ac.jp/en/organization)
+  in the combined analysis of data collected from their detectors. These data consist of gravitational-wave
+  data collected at any of the four interferometers but with simulated signals, as well as some other datasets,
+  used for data analysis purposes in detecting black hole collisions and other cosmic phenomena as
+  part of the NSF's multi-messenger astronomy initiatives.
 
-organization: # Name of the organization
-dataVisibility: # public, or private
+  These data are not yet released to the public.
+organization: California Institute of Technology
+dataVisibility: private
 size: # In bytes so I can parse how I want
+objectCount: 1,000,000
```

Comment:
Can you remove the commas? Not sure they'll be interpreted correctly.

Reply:
Just made that commit